make-util.js

```diff
@@ -1133,4 +1133,45 @@ var getTaskNodeVersion = function(buildPath, taskName) {
 }
 exports.getTaskNodeVersion = getTaskNodeVersion;
 
+var toOverrideString = function(object) {
+    return JSON.stringify(object).replace(/"/g, '\\"');
+}
+
+exports.toOverrideString = toOverrideString;
+
+var createExtension = function(manifest, publish) {
+    console.log('Installing task dependencies...');
+
+    var rootPath = process.cwd(); 
+    var tasksPath = path.join(rootPath, 'Tasks');
+    var tasks = fs.readdirSync(tasksPath);
+    console.log(tasks.length + ' tasks found.')
+    tasks.forEach(function(task) {
+        console.log('Processing task ' + task);
+        cd(path.join(tasksPath,task));
+
+        console.log('Installing PRODUCTION npm dependencies for task (' + task + ')...');
+
+        run('npm install --only=prod');
```

Comment:
I don't think we need to do `npm install` inside `Tasks/` anymore - looks like an ancient artifact. Could you please check if we can remove this?

Reply:
Yeah, I removed this and I tested changes, everything seems to be working fine.